### ISSUE TITLE
USER ISSUE: No Rabbits (propably invisible)

### ISSUE THREAD

**Issue by stuck1a** [StrangeLoopGames/EcoIssues#16537]
**Version:** 0.8.3.3 beta

**Steps to Reproduce:**
Can't say for sure. It happened alot, since I saw the last rabbit in this world, but I could see some at the very beginning. I'm pretty sure, it is just unable to load or show its sprites. Same for fox.

**Expected behavior:**
There should have been any rabbit within its current population of 29.

**Actual behavior:**
Instead I couldn't find any even after 2 hours of searching in all of their population areas.

**Do you have mods installed? Does issue happen when no mods are installed?:**
No mods, local server (singleplayer), default settings.

**Comment by SlayksWood** [comment]
@stuck1a Thank you for the report. I need some more information.
1. Provide graphics settings. (You can provide screenshot).
Pay your attention to the parameter 'Level of details'. If you have it at a low value than Hares could run away before you can see them.(screenshot from 0.9.0):
![image](https://user-images.githubusercontent.com/45708377/83396820-3b9a9680-a405-11ea-89cd-9bfc71b64762.png)
2. Provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by stuck1a** [comment]
@SlayksWood You'r absolutely right, its a caused by my bad performance or rather by my graphic settings. I've focused on the "view distance" setting, so I excluded this cause too early.
After setting the PlayerFlee-Attribute of hare's to false, I was able to find some. They will appear, if I approach to a distance of one block or less.
![GraphicSettings_stuck1a](https://user-images.githubusercontent.com/60481324/83493438-149b9d80-a4b5-11ea-96d3-337c76ba4d70.png)

I've added the logs for the case its still relevant (there are some catched NullReferenceExpetions, but I didn't mention any of them in-game)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/4715482/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/4715483/Player-prev.log)
